### ISSUE TITLE
Reduce APK size

### ISSUE THREAD

**Issue by xlogix** [krsnvijay/KOOC#2]
The drawables used in the project have a heavy footprint.

**Comment by krsnvijay** [comment]
yes , I've fixed it in the next version.

**Comment by xlogix** [comment]
You will? Can you commit that ASAP?

**Comment by krsnvijay** [comment]
After ct1

**Comment by xlogix** [comment]
![buildtime](https://cloud.githubusercontent.com/assets/16256275/23576689/3baf4e2e-00d2-11e7-9db9-286d2fa07298.JPG)

**Comment by krsnvijay** [comment]
Lol  it took that long because of the png optimizer plugin in build.gradle

**Comment by xlogix** [comment]
Yes, I saw that in gradle console. But, png files are more optimised for Android. I'll try reducing their size and format.
